#166239312 Set up Heroku deployment

Emmanuel-Dominic opened the pull request:
#### What does this PR do?
 - setup and deploy Heroku plus enabling review apps

#### Description of Task to be completed?
 - Currently, we do not have any deployment of the application. There is a need to deploy the current state of the application on Heroku so as to monitor progress and have visibility of the working system.

#### How should this be manually tested?
 - Simply click on this link to access the Heroku application URL [Heroku Link](https://wger-kronos.herokuapp.com)

#### What are the relevant pivotal tracker stories?
- [#166239312](https://www.pivotaltracker.com/n/projects/2350327/stories/166239312)

Rhytah commented:
@Emmanuel-Dominic  could you consider editing the PR description, especially `What does this PR do?` to be more specific to the story scope of the task at hand.

reifred reviewed (changes requested):
Thanks for the work done @Emmanuel-Dominic, kindly include the links in the PR description for us to test the deployments you have set up. In addition, make your story id linkable to pivotal tracker.

Manorlds-Eaglespark reviewed (commented):
Thanks for this great work so far, however, to approve your changes, I will wait till all the tests are passing on Travis.

AGMETEOR commented:
@Emmanuel-Dominic change the description to try and match the story description on the PT Board. Also, link the story id to the PT story.

AGMETEOR commented on settings.py:
```diff
@@ -0,0 +1,65 @@
+#         'HOST': os.environ.get('HOST'),
+#         'PORT': os.environ.get('PORT'),
+#     }
+# }
```
Is this commented out code necessary?

Emmanuel-Dominic committed:
chore(heroku) setup heroku deployment

 - Deploy the current state of the application on heroku.
 - Monitor progress and have visibility of the working system.

[Finishes #166239312]
